src/lib/state.ml

```diff
@@ -1,21 +1,26 @@
 
 type t = Sqlite3.db
 
-let initializers = ref []
+let initializers = ref [[]; []; []; [];[]; []; []]
```

Comment:
I don't think we need all the priority stuff. I agree that currently it may be a bit broken, mainly because there isn't enough code for all modules to be 1) linked when building the actual binary, and 2) have the proper dependencies between modules so that tables are initialized in the correct order. These two points should be fixed in PR #51 ; I'm very sorry for the oversight when reviewing (and implementing) previous PRs.